h2o-genmodel/src/main/java/hex/genmodel/algos/targetencoder/EncodingMaps.java

```diff
@@ -0,0 +1,55 @@
+package hex.genmodel.algos.targetencoder;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class EncodingMaps {
```

Comment:
Not serializable :(

Reply:
@michalkurka Yeah... I fixed this but put it into Mojo support PR https://github.com/h2oai/h2o-3/pull/3282/commits/7d7d04c23b3221dd7c4fdddc39a89c3492b27f9f. Will port those changes here.

Reply:
@michalkurka ported